docs/index.rst

```diff
@@ -1,19 +1,19 @@
-Confit: Painless Configuration
+Confuse: Painless Configuration
 ==============================
 
-`Confit`_ (*con-FEE*) is a straightforward, full-featured configuration system
+`Confuse`_ (*con-FEE*) is a straightforward, full-featured configuration system
```

Comment:
haha, the pronunciation should probably be removed. I just did a couple of
s//g s and ensured the tests still passed :P

On Thu, Apr 28, 2016 at 11:33 AM, Jack Wilsdon <notifications@github.com>
wrote:

> In docs/index.rst
> <https://github.com/sampsyo/confit/pull/21#discussion_r61449301>:
>
> >  ==============================
> >
> > -`Confit`_ (*con-FEE*) is a straightforward, full-featured configuration system
> > +`Confuse`_ (*con-FEE*) is a straightforward, full-featured configuration system
>
> I'm not sure if the pronunciation would stay as *con-FEE*, as that isn't
> anywhere near how you pronounce Confuse 😆.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/sampsyo/confit/pull/21/files/2d761ed72ddbfc0fb6cb5f7700cde769ef1b5ae1#r61449301>
>



-- 
have a nice day
-jck